devito/core/cpu.py

```diff
@@ -111,7 +110,6 @@ def _specialize_clusters(cls, clusters, **kwargs):
         # turn may enable further optimizations
         clusters = fuse(clusters)
         clusters = eliminate_arrays(clusters, template)
-        clusters = scalarize(clusters, template)
```

Comment:
note for reviewers: this pass now unnecessary, as everything is done naturally inside cire()